pkg/lib/manifest2po

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/python3
+# This file is part of Cockpit.
+#
+# Copyright (C) 2025 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import argparse
+import collections
+import json
+import pathlib
+import re
+from typing import Any, DefaultDict, Dict, Iterable, List, Set
+
+PO_HEADER = """msgid ""
```

Comment:
Make this an `r"""` string to avoid needing to double-escape the `\\n`.

Reply:
I could but would require a ruff ignore "f-string without placeholder" so its more a style thing

Reply:
`r"""`, not `f"""`.